tests/unit/test-cmd/test.build.js

```diff
@@ -28,37 +28,39 @@ describe('build', () => {
 
     return withTempDir(
       (tmpDir) =>
-        build({
-          sourceDir: fixturePath('minimal-web-ext'),
-          artifactsDir: tmpDir.path(),
-        })
-        .then((buildResult) => {
-          assert.match(buildResult.extensionPath,
-                       /minimal_extension-1\.0\.zip$/);
-          return buildResult.extensionPath;
-        })
-        .then((extensionPath) => zipFile.open(extensionPath))
-        .then(() => zipFile.extractFilenames())
-        .then((fileNames) => {
-          fileNames.sort();
-          assert.deepEqual(fileNames,
-                           ['background-script.js', 'manifest.json']);
-        })
+        build(
```

Comment:
Nit, for chains of promises I usually opt for the following style (which seems to be compatible with the eslint rules enforced in this branch, but, as the above, it is a matter of taste and it is not a mandatory required change for this review):

```js
    return withTempDir(
      (tmpDir) =>
        build({
          sourceDir: fixturePath('minimal-web-ext'),
          artifactsDir: tmpDir.path(),
        }).then((buildResult) => {
          assert.match(buildResult.extensionPath,
                       /minimal_extension-1\.0\.zip$/);
          return buildResult.extensionPath;
        }).then(
          (extensionPath) => zipFile.open(extensionPath)
        ).then(
          () => zipFile.extractFilenames()
        ).then((fileNames) => {
          fileNames.sort();
          assert.deepEqual(fileNames,
                           ['background-script.js', 'manifest.json']);
        })
    );
```